changelog/_unreleased/2025-03-20-move-notifications-from-admin-to-core.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Move notifications from admin to core
+---
+# Core
+* Added `Shopware\Core\Framework\Notification\NotificationCollection`, `Shopware\Core\Framework\Notification\NotificationDefinition` & `Shopware\Core\Framework\Notification\NotificationEntity`
```

Comment:
Do we need that file at all now?

Reply:
I added a section to 6.7 upgrade core and removed this changelog